public/app/features/integrated-alerting/components/AlertRules/AlertRulesActions/AlertRulesActions.test.tsx

```diff
@@ -4,16 +4,86 @@ import { act } from 'react-dom/test-utils';
 import { dataQa } from '@percona/platform-core';
 import { AlertRulesActions } from './AlertRulesActions';
 import { AlertRulesProvider } from '../AlertRules.provider';
-import { formattedRulesStubs } from '../__mocks__/alertRulesStubs';
+import { rulesStubs, formattedRulesStubs } from '../__mocks__/alertRulesStubs';
+import { AlertRulesService } from '../AlertRules.service';
+import { Messages } from './AlertRulesActions.messages';
 
 jest.mock('../AlertRules.service');
 jest.mock('app/core/app_events');
 
+const alertRulesServiceCreate = jest.spyOn(AlertRulesService, 'create');
+
 describe('AlertRulesActions', () => {
   afterEach(() => {
     jest.clearAllMocks();
   });
 
+  it('calls the API to crate an alert rule on copy', async () => {
+    const testRule = rulesStubs[1];
+
+    const wrapper = mount(<AlertRulesActions alertRule={formattedRulesStubs[1]} />);
+
+    const expectedResult = {
+      ...testRule,
+      disabled: true,
+      summary: `${Messages.copyOf} ${testRule.summary}`,
+      custom_labels: undefined as any,
+      channel_ids: ['test_ch'],
+      template_name: 'test 2',
+    };
+
+    await act(async () => {
+      wrapper
+        .find(dataQa('copy-alert-rule-button'))
+        .at(0)
+        .simulate('click');
+    });
+
+    expect(alertRulesServiceCreate).toBeCalledTimes(1);
+    expect(alertRulesServiceCreate.mock.calls[0][0]).toEqual(expectedResult);
```

Comment:
wouldn't `expect(alertRulesServiceCreate).toHaveBeenCalledWith(expectedResult)` work here?

Reply:
Yes, good catch, I forgot to update this after a test :)